docs/en/rules/Azure.Automation.ManagedIdentity.md

```diff
@@ -0,0 +1,97 @@
+---
+severity: Important
+pillar: Security
+category: Identity and access management
+resource: Automation Service
```

Comment:
Wondering if we should use **Automation Account** for resource?

Reply:
@ArmaanMcleod Yes, probably a good idea to change this.